### PR TITLE
Add landing page

### PR DESCRIPTION
**Pull request by AlejandroPerezMartin**

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -46,19 +46,36 @@ module.exports.tmpLogin = {
   }
 };
 
-module.exports.main = {
+module.exports.repos = {
   method: 'GET',
-  path: '/',
+  path: '/repos',
   config: {
     auth: 'session'
```
Comment (Antonio-Laguna) [inline routes/index.js]: I don't think this should be behind auth.

File: web/views/pages/landing/styles.css [hunk 2]
```diff
@@ -0,0 +1,208 @@
+
+header {
+  background: linear-gradient(224deg, var(--color-accent1), var(--color-accent2));
+  background-size: 400% 400%;
+  -webkit-animation: movingGradient 30s ease infinite;
+  -moz-animation: movingGradient 30s ease infinite;
+  animation: movingGradient 30s ease infinite;
```
Comment (Antonio-Laguna) [inline web/views/pages/landing/styles.css]: We don't need to do any prefixing.

File: web/views/pages/landing/styles.css [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+    background: var(--color-white);
+    color: var(--color-accent1);
+
+    & svg {
+      fill: var(--color-accent1);
+    }
+  }
+
+  & svg {
+    fill: var(--color-white);
```
Comment (Antonio-Laguna) [inline web/views/pages/landing/styles.css]: Might use here `currentColor` and avoid the change on hover

File: web/views/pages/landing/styles.css [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+    0%{background-position:90% 0%}
+    50%{background-position:11% 100%}
+    100%{background-position:90% 0%}
+}
+@-moz-keyframes movingGradient {
+    0%{background-position:90% 0%}
+    50%{background-position:11% 100%}
+    100%{background-position:90% 0%}
+}
+@keyframes movingGradient {
```
Comment (Antonio-Laguna) [inline web/views/pages/landing/styles.css]: No prefixing here.  Ensure spacing for this is the same as `slideInFromBottom` below

File: routes/index.js (hunk 1 above)
Comment (jgallen23) [reply routes/index.js]: I think this has to be behind auth, right?

**Pull request merged**